case-lib/lib.sh

```diff
@@ -752,17 +752,37 @@ func_lib_check_pa()
 # However, 1. arrays would complicate the user interface 2. ALSA does not
 # seem to need arguments with whitespace or globbing characters.
 
+channel=$(func_pipeline_parse_value "$idx" channel)
+rate=$(func_pipeline_parse_value "$idx" rate)
+fmt_elem=$(func_pipeline_parse_value "$idx" fmt_elem)
+dev=$(func_pipeline_parse_value "$idx" dev)
+duration=$(func_pipeline_parse_value "$idx" duration)
+
 aplay_opts()
 {
-    dlogc "aplay $SOF_ALSA_OPTS $SOF_APLAY_OPTS $*"
-    # shellcheck disable=SC2086
-    aplay $SOF_ALSA_OPTS $SOF_APLAY_OPTS "$@"
+    if [[ $SOF_TINY_ALSA == true ]]; then
```

Comment:
just few additional suggestions from my side:
1.
```suggestion
    if [[ "$SOF_TINY_ALSA" = true ]]; then
```

2. `SOF_TINY_ALSA` - I'm not sure it is "scalable" enough as a bool variable in comparison to `SOF_ALSA_TOOL=tinyalsa` or whatever other/proprietary tool, a custom script tool etc. it might be in the future. 

3. anyway, document it: 
* [in comments](https://github.com/thesofproject/sof-test/blob/main/case-lib/lib.sh#L727-L753)
* [in README.md](https://github.com/thesofproject/sof-test/blob/main/README.md?plain=1#L56)

4. add `dlogc` similar to what the alsa branch does

Reply:
Regarding point number 2, I think that it will always force the use of the SOF_ALSA_TOOL parameter, e.g., SOF_ALSA_TOOL=alsa.

Reply:
> SOF_ALSA_TOOL=alsa, which could break backward compatibility.

This can be a default value. Default values are often implemented like this:

```shell
: ${SOF_ALSA_TOOL:=alsa}
```

https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

Word expansion -> parameter expansion -> default value.